### ISSUE TITLE
Meeting 5 - Constraints update  and Challenge example.

### ISSUE THREAD

**Issue by misik-eng** [MIT-Emerging-Talent/ET6-foundations-group-05#10]
Meeting 5- Constraints and Challenge Example.
Thursday, December 26 · 6:00 – 7:30pm
Time zone: America/New_York
Google Meet joining info
Video call link: https://meet.google.com/ynr-bwkz-sek

- [x] One of us will create a new branch for Constraints, update our Constraints, commit the changes to Constraints branch, and then push it to our remote group repository. After that, we will create a pull request, and one of us will review and merge it.

- [x] We will check our [Project Board](https://github.com/orgs/MIT-Emerging-Talent/projects/105/views/1).
  
- [x] We will read this [Development Guide](https://github.com/MIT-Emerging-Talent/ET6-foundations-group-05/blob/main/collaboration/guide/6_development.md) and try to learn how should we proceed with development steps.

- [x] One of us (I hope Gennadii)  will demonstrate us how to find a challenge and work on it. (If we have time & don't worry about the Google Meet time restriction. Now we can hold meetings for up to 24 hours. ).

- [x] We will decide on our next meeting. Thank you, everyone, for being active and participating in all of our meetings. This is the best team ever. Go Shining Stars!

**Comment by doctorbanu** [comment]
I have some commitments tomorrow, but I will try to adjust them. I am not sure if I can make it or not, but I will do my best to meet you. Thank you, guys. You are the best.🧚🏼‍♀️

**Comment by imwaymaran** [comment]
We had our fifth meeting on Thursday, December 26. Muhammet worked on creating a new branch for the Constraints, updated the document, committed the changes, pushed them to the remote repository, and completed the process with a pull request review and merge. We also reviewed our project board and discussed the Development Guide to plan our next steps. I demonstrated how to find and work on a challenge. Closing this issue now.
